oap-server/server-configuration/configuration-api/src/main/java/org/apache/skywalking/oap/server/configuration/api/ConfigWatcherRegister.java

```diff
@@ -110,6 +110,67 @@ void configSync() {
 
             LOGGER.trace("Current configurations after the sync." + LINE_SEPARATOR + register.toString());
         });
+
+        GroupConfigsSync(configTable);
+    }
+
+    private void GroupConfigsSync(Optional<ConfigTable> configTable) {
+        configTable.ifPresent(config -> {
+            config.getGroupItems().forEach(groupConfigItems -> {
+                String groupConfigItemName = groupConfigItems.getName();
+                WatcherHolder holder = register.get(groupConfigItemName);
+                if (holder != null) {
```

Comment:
Use fast return to reduce the nested level

```suggestion
                if (holder == null) {
                    return;
                }
```